addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -746,8 +746,9 @@ export class OdooEditor extends EventTarget {
 
         // sanitize and mark current position as sanitized
         sanitize(target);
-        this._pluginCall('sanitizeElement',
-                         [target.parentElement]);
+        const sanitizeElement = target.parentElement ?
+                                target.parentElement : target;
```

Comment:
Unless you are particularly fond of the current version, `A ? A : B` is equivalent to `A || B` so I suggest `target.parentElement || target`.

Reply:
sure